pkg/network/http/batch_manager.go

```diff
@@ -36,25 +36,26 @@ type batchManager struct {
 	numCPUs    int
 }
 
-func newBatchManager(batchMap, batchStateMap *ebpf.Map, numCPUs int) *batchManager {
+func newBatchManager(batchMap, batchStateMap *ebpf.Map, numCPUs int) (*batchManager, error) {
```

Comment:
We don't need `batchStateMap` as an argument anymore, right?

Reply:
aha, good catch